src/Bridges/bridgeoptimizer.jl

```diff
@@ -87,7 +87,7 @@ function MOI.supports(b::AbstractBridgeOptimizer,
     return MOI.supports(b.model, attr)
 end
 function MOI.copy_to(b::AbstractBridgeOptimizer, src::MOI.ModelLike;
-                   copynames=false)
+                   copynames = true)
```

Comment:
copy_names?

Reply:
Yes, we should rename that too, I planned to do this in the next PR